docs/source/user-guide/tasks/manage-environments.rst

```diff
@@ -123,6 +123,46 @@ Use the terminal for the following steps:
 
    You can also use ``conda info --envs``.
 
+.. _specifying-environment-platform:
+
+Specifying a different target platform for an environment
+=========================================================
+
+By default, ``conda`` will create environments targeting the platform
+currently running on. You can check which one is yours by running
+``conda info`` and checking the ``platform`` entry.
+
+However, in some cases you might want to create an environment for a
+different target platform or architecture. To do so, use the
+``--platform`` flag available in the ``conda create`` and
+``conda env create`` commands. The value must be a valid platform.
+
+For example, a user running macOS on the Apple Silicon platform
+might want to create a ``python`` environment for Intel processors
+and emulate the executables with Rosetta. The command would be:
+
+.. code::
+
+   conda create --platform osx-64 --name python-x64 python
+
+Note that you can't specify this flag for existing environments.
+The environment will be annotated with the custom configuration and
+subsequent operations on it will remember the target platform.
```

Comment:
```suggestion
.. note::

   You can't specify this flag for existing environments.
   The environment will be annotated with the custom configuration and
   subsequent operations on it will remember the target platform.
```

Maybe this would look better in an admonition?

Reply:
I agree with the admonition. I also have a few rewrite suggestions.

```suggestion
.. note::
   You can't specify the ``--platform`` flag for existing environments. 
   When created, the environment will be annotated with the custom configuration and
   subsequent operations on it will remember the target platform.
```
Just to be a bit more clear there.